app/cafe.py

```diff
@@ -0,0 +1,18 @@
+from datetime import date
+from app.errors import (NotVaccinatedError,
+                        NotWearingMaskError,
+                        OutdatedVaccineError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | Exception:
```

Comment:
There is no need to indicate in the return type annotation that errors could be raised, it is redundant. Remove it, please

Reply:
done